attributes/default.rb

```diff
@@ -91,6 +91,10 @@
 default['datadog']['yumrepo_proxy_password'] = nil
 default['datadog']['windows_agent_url'] = 'https://s3.amazonaws.com/ddagent-windows-stable/'
 
+# Location of additional rpm gpgkey to import (with signature `e09422b3`). In the future the rpm packages
+# of the Agent will be signed with this key.
+default['datadog']['yumrepo_gpgkey_new'] = "#{yum_protocol}://yum.datadoghq.com/DATADOG_RPM_KEY.public.E09422B3"
```

Comment:
@yannmh: the new rpm key currently lives at this URL, I'm not a huge fan of its name though, I'd prefer something like `DATADOG_RPM_KEY_E09422B3.public`, what do you think?